packages/playground/src/dashboard/components/user_farms.vue

```diff
@@ -235,7 +236,11 @@ export default {
         farmsCount.value = count || filteredFarms.length;
       } catch (error) {
         console.log(error);
-        createCustomToast("Failed to get user farms!", ToastType.danger);
+        if (error instanceof ConnectionError) {
+          createCustomToast("Failed to get farms. Please check your connection.", ToastType.danger);
+        } else {
+          createCustomToast("Failed to get farms.", ToastType.danger);
```

Comment:
i got this . can you please add "Please check your connection. " here too?
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/56790126/19673ce7-164b-461e-8dfc-027d5d83db7f)

Reply:
If i added same msg , what is the point then of checking on error type ?

Reply:
This issue should be fixed here https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1735 @AlaaElattar @amiraabouhadid

Reply:
@amiraabouhadid Another error type not connection related might happen in the else, so I think it's better this way.

Reply:
I checked #1735 and it indeed fixes the issue in case no connection is needed.